src/wrappers/express/ExpressWrapper.ts

```diff
@@ -0,0 +1,43 @@
+import Reporter from '../../Reporter';
+import ConfigProvider from '../../config/ConfigProvider';
+import { ApplicationManager } from '../../application/ApplicationManager';
+import ExecutionContextManager from '../../context/ExecutionContextManager';
+import * as ExpressExecutor from './ExpressExecutor';
+import WrapperUtils from '../WebWrapperUtils';
+import ExecutionContext from '../../context/ExecutionContext';
+import ThundraLogger from '../../ThundraLogger';
+
+export function expressMW(opts: any = {}) {
+    ApplicationManager.setApplicationInfoProvider().update({
+        applicationClassName: 'Express',
+        applicationDomainName: 'API',
+    });
+
+    const config = opts.config || ConfigProvider.thundraConfig;
+    const { apiKey } = config;
+    const reporter = opts.reporter || new Reporter(apiKey);
+    const pluginContext = opts.pluginContext || WrapperUtils.createPluginContext(apiKey, ExpressExecutor);
+    const plugins = opts.plugins || WrapperUtils.createPlugins(config, pluginContext);
+
+    if (!opts.disableAsyncContextManager) {
+        WrapperUtils.initAsyncContextManager();
+    }
+
+    return (req: any, res: any, next: any) => ExecutionContextManager.runWithContext(
+        WrapperUtils.createExecContext,
+        async function () {
+            const context: ExecutionContext = this;
+            try {
+                await WrapperUtils.beforeRequest(req, plugins);
+                res.once('finish', () => {
+                    ExecutionContextManager.set(context);
+                    WrapperUtils.afterRequest(res, plugins, reporter);
+                });
+            } catch (err) {
+                ThundraLogger.error('<ExpressWrapper> Error occured in in ExpressWrapper: ', err);
```

Comment:
- There are multiple `in` in the log message
- No need to extra space after `:` as space is already added automatically after each argument